src/plugins/discourse/mirrorRepository.js

```diff
@@ -60,6 +60,18 @@ export interface ReadRepository {
   topicsInCategories(
     categoryIds: $ReadOnlyArray<CategoryId>
   ): $ReadOnlyArray<TopicId>;
+
+  /**
+   * Gets a Topic by ID.
+   */
+  topicById(id: TopicId): ?Topic;
+
+  /**
+   * Gets a number of Posts in a given Topic.
+   *
+   * Note: starting with the first post, ordered by `indexWithinTopic`.
```

Comment:
Please update docString to explain what the `numberOfPosts` argument does.

Reply:
Done, updated to

```js
  /**
   * Gets a number of Posts in a given Topic. Starting with the first post,
   * ordered by `indexWithinTopic`.
   *
   * numberOfPosts: the maximum number of posts to get (may return fewer).
   */
  postsInTopic(topicId: TopicId, numberOfPosts: number): $ReadOnlyArray<Post>;
```